### PR TITLE
Alma support bugfixes

### LINKED ISSUE #11: Wrong check on volumes
The list of volumes mounted being checked is wrong. 
Please remove these and provide the minimum viable 

```
 * || check_fs_used /dev 90%
 * || check_fs_used / 90%
 * || check_fs_used /boot/efi 90%
 * || check_fs_used /mnt 90%
 * || check_fs_used /sched 90% => This doesn't exists
 * || check_fs_used /shared 90% => This doesn't exists
 * || check_fs_used /mnt/resource_nvme 90% => This doesn't exists

```

### PR DESCRIPTION
**Pull request by rafsalas19**
- Addressing mnt checks for non-existent directories
- Making NDv4 conf files more uniform
- Modify install scripts to work for Alma

### CODE REVIEW

File: conf/nd96amsr_v4.conf [hunk 1]
```diff
@@ -28,7 +28,7 @@
  * || check_fs_mount_rw -t "securityfs" -s "securityfs" -f "/sys/kernel/security"
  * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/dev/shm"
  * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/run/lock"
- * || check_fs_mount_ro -t "tmpfs" -s "tmpfs" -f "/sys/fs/cgroup"
+ # * || check_fs_mount_ro -t "tmpfs" -s "tmpfs" -f "/sys/fs/cgroup"
```
Comment (jithinjosepkl) [inline conf/nd96amsr_v4.conf]: Any value in keeping this as commented?

File: conf/nd96asr_v4.conf [hunk 2]
```diff
@@ -8,10 +8,39 @@
 # Fri Jan 14 21:17:09 UTC 2022
 #
 
+#######################################################################
```
Comment (jithinjosepkl) [inline conf/nd96asr_v4.conf:11]: do you think we can unify the conf files for NDv4 40 and 80 GB SKU's?

Comment (rafsalas19) [reply conf/nd96asr_v4.conf:11]: The reason they are separate is due to differences in the expected values of "check_hw_physmem" tests. Ndv4 80G has more main memory.

The tests:
80G sku: * || check_hw_physmem 1857830712kB 1857830712kB 3%
40G sku:  * || check_hw_physmem 928858388kB 928858388kB 3%

aside from this single test the files are pretty much the same.

The options are:
- Keep individual files
- Unify them and instruct the user to modify the single test based on sku

File: conf/nd96amsr_v4.conf (hunk 1 above)
Comment (rafsalas19) [reply conf/nd96amsr_v4.conf]: Let me address

Comment (rafsalas19) [reply conf/nd96amsr_v4.conf]: adressed

File: conf/nd96amsr_v4.conf [hunk 3]
```diff
@@ -24,37 +24,37 @@
...
+ * || check_fs_mount_rw -t "debugfs" -s "debugfs" -f "/sys/kernel/debug"
  * || check_fs_mount_rw -t "hugetlbfs" -s "hugetlbfs" -f "/dev/hugepages"
  * || check_fs_mount_rw -t "mqueue" -s "mqueue" -f "/dev/mqueue"
- * || check_fs_mount_rw -t "debugfs" -s "debugfs" -f "/sys/kernel/debug"
  * || check_fs_mount_rw -t "vfat" -s "/dev/sda15" -f "/boot/efi"
  * || check_fs_mount_rw -t "fusectl" -s "fusectl" -f "/sys/fs/fuse/connections"
  * || check_fs_mount_rw -t "configfs" -s "configfs" -f "/sys/kernel/config"
  * || check_fs_mount_rw -t "ext4" -s "/dev/sdb1" -f "/mnt"
  * || check_fs_mount_rw -t "tracefs" -s "tracefs" -f "/sys/kernel/debug/tracing"
+#  * || check_fs_mount_rw -t "nfs4" -s "*:/*" -f "/sched"
```
Comment (moes1) [inline conf/nd96amsr_v4.conf:39]: Why are we adding these tests that are then commented out in both nd96 conf files?

File: conf/nd96amsr_v4.conf [hunk 4]
```diff
@@ -24,37 +24,37 @@
  * || check_fs_mount_rw -t "devtmpfs" -s "devtmpfs" -f "/dev"
  * || check_fs_mount_rw -t "devpts" -s "devpts" -f "/dev/pts"
  * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/run"
- * || check_fs_mount_rw -t "ext4" -s "/dev/root" -f "/"
```
Comment (rajvi) [inline conf/nd96amsr_v4.conf:27]: Are the tests being deleted not needed?

**Review by rajvi — APPROVED** [review]

File: conf/nd96amsr_v4.conf (hunk 3 above)
Comment (rafsalas19) [reply conf/nd96amsr_v4.conf:39]: These are common with Cyclecloud. So I thought we should leave them there in the event a customer using cycle cloud would like to uncomment. Thoughts?

File: conf/nd96asr_v4.conf (hunk 2 above)
Comment (rafsalas19) [reply conf/nd96asr_v4.conf:11]: I will keep separate for now

File: conf/nd96amsr_v4.conf (hunk 3 above)
Comment (rafsalas19) [reply conf/nd96amsr_v4.conf:39]: I will leave for now

File: conf/nd96amsr_v4.conf (hunk 4 above)
Comment (rafsalas19) [reply conf/nd96amsr_v4.conf:27]: in short yes. They don't add much value and are not uniform across all OS's

**Pull request merged**